libs.freemarker/external/binaries-list

```diff
@@ -14,4 +14,4 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-5EEAAC41164FEBCB79C73BEBD678A7B6C10C3E80 freemarker-2.3.19.jar
+A251045E5FADD02824D17F1AA8C412ACCF1AA1C9 org.freemarker:freemarker:2.3.19
```

Comment:
This is in the license:

> 2.  The end-user documentation included with the redistribution, if any, must
>     include the following acknowlegement:
>       "This product includes software developed by the
>       Visigoth Software Society (http://www.visigoths.org/)."
>     Alternately, this acknowlegement may appear in the software itself, if and
> wherever such third-party acknowlegements normally appear.

I would this include a "-notice" file, that in the end should end up in the apache NOTICE file. From my reading this is similar to the apache NOTICE file, that needs to be redistributed.